plugins/catalog/package.json

```diff
@@ -30,6 +30,7 @@
     "@material-ui/core": "^4.9.1",
     "@material-ui/icons": "^4.9.1",
     "@material-ui/lab": "4.0.0-alpha.45",
+    "moment": "^2.26.0",
```

Comment:
Are we doing moment and not date-fns for example? Talking about bundle size ;)

Reply:
We've kinda just given up and gone with moment. Although tbh big static dependencies aren't that bad given our bundle splitting strategy.